osf/management/commands/sync_doi_metadata.py

```diff
@@ -69,6 +69,42 @@ def sync_doi_metadata(modified_date, batch_size=100, dry_run=True, sync_private=
             sync_identifier_doi.apply_async(kwargs={'identifier_id': identifier.id})
 
 
+@app.task(name='osf.management.commands.sync_doi_empty_metadata_dataarchive_registrations_command', max_retries=5, default_retry_delay=RATE_LIMIT_RETRY_DELAY)
+def sync_doi_empty_metadata_dataarchive_registrations(modified_date, batch_size=100, dry_run=True, sync_private=False, rate_limit=100):
+    identifiers = Identifier.objects.filter(
+        category='doi',
+        deleted__isnull=True,
+        modified__lte=modified_date,
+        object_id__isnull=False,
+        content_type_id=ContentType.objects.get_for_model(Registration).id
+    )
```

Comment:
I don't think we want to start from the identifiers, here. We have a lot of registrations and a lot of identifiers on Production, but we'll have almost no registrations in the data archive provider. I think we want to start with the registrations that are in the data archive provider, then grab the identifiers from there.